### DISCUSSION TITLE
Migrating from DNN to Oqtane

### DISCUSSION THREAD

**Discussion by mortezashoghi** [oqtane/oqtane.framework#1948] [Q&A] [answered]
Our company is used DNN more than 10 years .at now we are inspecting Oqtane framework .

**In your opinion** , do you think that migrating from DNN to Oqtane will be useful?

**Comment by nicpitsch** [comment, marked as answer]
The [Oqtane Blog](https://www.oqtane.org/Resources/Blog) contains some posts related to DNN. Keep in mind that [Oqtane is a modular application framework - it is not a content management system (CMS)](https://www.oqtane.org/Resources/Blog/PostId/538/oqtane-philosophy). You could also say it is more than a CMS only. 

If you want to use it as a CMS, I would recommend to try it together e.g. with [2sxc custom Module](https://github.com/oqtane/oqtane.framework/discussions/1894), look this Blog post [announcement about 2sxc 12 Hybrid for Dnn and Oqtane](https://2sxc.org/en/blog/post/2sxc-12-hybrid-for-dnn-and-oqtane-released) too. [Leigh Pointer](https://github.com/oqtane/oqtane.framework/discussions?discussions_q=author%3Aleigh-pointer), as another example, is also active concerning Module developent, i'm not sure whether they are public available.
